test-cases/tests/performance/k04-run-performance-test-against-rhoam.md

```diff
@@ -0,0 +1,98 @@
+
+- create a `perf-test-start-time.txt` file as described in [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh).
+  - the actual performance test run doesn't start immediately, the performance test suite creates various 3scale (Product, Backend, Application, Application Plan) and SSO (realm, client, users) entities first
+  - best to track the log of Hyperfoil controller to get the exact time the when `rampUp` phase starts
+
+8. Create a `perf-test-end-time.txt` for [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh)
+
+- to get the exact time track the Hyperfoil controller log
+
+9. Collect the data about the performance test run
+
+- from [alerts-during-perf-testing](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/alerts-during-perf-testing.sh)
+- from Hyperfoil
+  - install Hyperfoil locally
+  - bin/cli.sh
+  - connect <hyperfoil-url-without-protocol> -p <port-8090-is-default>
+  - runs # to see all the runs
+  - status <your-run-name>
+  - stats <your-run-name>
+  - export -f json -d . <your-run-name> # to export the data
+  - use [report tool](https://github.com/Hyperfoil/report) to generate the HTML out of the exported data
+- eye review of various Grafana Dashboards, see [this guide](https://docs.google.com/document/d/1KznoB-we73lGUViJApVHyBoIgh3xpgyak6ODAEAHbwk/edit?usp=sharing) on how to do it
+- use [capture_resource_metrics script](https://github.com/integr8ly/integreatly-operator/blob/master/scripts/capture_resource_metrics.sh) to get the data
+- create a spreadsheet with all the collected data
+  - see [example spreadsheet](https://docs.google.com/spreadsheets/d/1v_bZIk8B_thZi93hGBNiOOnbSix0gmpwy3LV_s4WFPw/edit?usp=sharing)
+- add information about the benchmark used
+  - the benchmark will be improved over the time so we need to take it into account when comparing results
+
+10. Analyse the results
+- compare with the previous runs
+
+11. Attach the spreadsheet to the JIRA ticket
+
+- store the spreadsheet in [Google Drive](https://drive.google.com/drive/folders/1DR2qV7Z2bbvJv47Vnah-n_Qm5lnlcQfd)
+- store the Hyperfoil report in Google Drive too
+- store the Grafana Dashboard screenshot there too if needed
```

Comment:
Screenshot? I though we would be taking snapshots...
A link to a snapshot can be added into spreadsheet with results and into JIRA.

Reply:
Yep, I typed `screenshot` despite thinking about `snapshot`. Will update

Reply:
fixed